s3contents/ipycompat.py

```diff
@@ -17,6 +17,7 @@
 from notebook.services.contents.filemanager import FileContentsManager
 from notebook.services.contents.manager import ContentsManager
 from notebook.services.contents.tests.test_contents_api import APITest
+from notebook.services.contents.tests.test_largefilemanager import TestLargeFileManager
```

Comment:
@ericdill this is failing right now - can you think of anything that might prevent this import from working? Feel like I'm missing something... It seems to be here: https://github.com/jupyter/notebook/tree/master/notebook/services/contents/tests

Reply:
version 4.* of the notebook didn't have this import. It was introduced in 5.0. This library tests for notebook 4.*, which I think it's time to drop. cc @danielfrg

Reply:
No problem dropping notebook 4.*